raster_loader/cli/snowflake.py

```diff
@@ -88,6 +95,17 @@ def upload(
         get_block_dims,
     )
 
+    if token is None and (username is None or password is None):
```

Comment:
It should probably also warn in case both user/pwd and token are provided simultaneously. It won't break, but only the token will be used, which could be confusing for the user.

Reply:
Right, on it